themes/bootstrap3/templates/RecordDriver/DefaultRecord/cover.phtml

```diff
@@ -10,6 +10,9 @@
     <div class="spinner"><i class="fa fa-spinner fa-spin"></i> <?=$this->translate('Loading')?>...</div>
     <div class="cover-container">
       <img <?php if ($linkPreview): ?>data-linkpreview="true" <?php endif; ?>data-recordsource="<?=$this->escapeHtmlAttr($driver->getSourceIdentifier())?>" data-recordid="<?=$this->escapeHtmlAttr($driver->getUniqueID())?>" data-coversize="<?=$this->escapeHtmlAttr($size)?>" class="recordcover ajax" alt="<?=$this->escapeHtmlAttr($alt); ?>" />
+      <p class="cover-source">
```

Comment:
Should this be hidden by default? If so, I'm not seeing the relevant styles, but maybe I'm missing something.

Reply:
Whole cover-container is hidden by default and ajax handler then displays parts which contains data